lib/src/widgets/animated_focus_light.dart

```diff
@@ -98,29 +98,28 @@ class AnimatedFocusLightState extends State<AnimatedFocusLight>
 
   @override
   Widget build(BuildContext context) {
-    return InkWell(
-      onTap: _targetFocus.enableOverlayTab
```

Comment:
Before, the whole screen was the inkwell and on top of the stack is positioned in the target a container with an InkWell, then when you tapped on the target the function `onTargetTap` was executed, when you tapped outside the target the function `onOverlayTap` was executed, this made that putting a conditional on whether or not to put the small InkWell in the target did not work for this purpose of making the target transparent to taps.